src/poll.rs

```diff
@@ -53,7 +119,32 @@ pub fn poll(fds: &mut [PollFd], timeout: libc::c_int) -> Result<libc::c_int> {
     Errno::result(res)
 }
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
+/// `ppoll()` allows an application to safely wait until either a file
```

Comment:
Distinguishing them would be good, I just wanted the description to be a little shorter so I grabbed it from the man page. Maybe instead say "Like `poll()` but can also wait for signals". How's that sound?